W-16945136 overview page updates (#244)
mulesoft/docs-partner-manager: modules/ROOT/pages/index.adoc

= Anypoint Partner Manager Overview
page-aliases: docs.mulesoft.com/partner-manager/latest

> Anypoint Partner Manager enables you to perform bidirectional Business-to-Business (B2B) message exchanges with your external business partner ecosystem. Partner Manager supports AS2, HTTP, HTTPS, SFTP, and FTP transport protocols. The supported message formats include CSV, EDIFACT, EDI X12, JSON, XML, X12 UCS, and X12 VICS.

Reviewer: ```suggestion
Perform bidirectional Business-to-Business (B2B) message exchanges with your external business partner ecosystem using Anypoint Partner Manager. Partner Manager supports AS2, HTTP, HTTPS, SFTP, and FTP transport protocols. Supported message formats include CSV, EDIFACT, EDI X12, JSON, XML, X12 UCS, and X12 VICS.
```
User value first

> Partner Manager unlocks a variety of use cases, such as order to cash, procure to pay, logistics planning, warehouse management and fulfillment workflows, and transportation execution.

Reviewer: ```suggestion
Partner Manager unlocks use cases such as order to cash, procure to pay, logistics planning, warehouse management and fulfillment workflows, and transportation execution.
```
